lightning/src/ln/channelmanager.rs

```diff
@@ -1610,6 +1641,32 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F
 		self.list_channels_with_filter(|&(_, ref channel)| channel.is_live())
 	}
 
+	/// Gets a list of payments in random order that are currently pending resolution.
+	///
+	/// This can be useful for figuring out whether or not a payment made needs to be retried.
+	/// In general, if it is not listed here, you should consider retrying it with a caveat: if a
+	/// payment is returned as [`RecentPaymentDetails::Fulfilled`], you may retry it, but should
+	/// not.
```

Comment:
Would it be simpler to say to "should consider retrying if not [`RecentPaymentDetails::Fulfilled`]"?